frontend/app/views/spree/checkout/_confirm.html.erb

```diff
@@ -6,6 +6,13 @@
 
 <br />
 
+<% if Spree::Config[:require_terms_and_conditions] %>
+  <p class="field" style='clear: both'>
+    <%= form.checkbox :terms_and_conditions %>
+    <%= form.label :terms_and_conditions, Spree.t(:accept_terms_and_conditions) %>
```

Comment:
You have to nest the checkbox into the label, otherwise you can't click the label and reach the checkbox: 
```
<% form.label :terms_and_conditions do %>
  <%= form.checkbox :terms_and_conditions do %>
    <%= Spree.t(:accept_terms_and_conditions) %>
  <% end %>
<% end %>
```

Semantically it's fine like you have it.